### ISSUE TITLE
Does IME work?

### ISSUE THREAD

**Issue by hajimehoshi** [fyne-io/fyne#63]
I tried to input Japanese text with Japanese IME, but I'm not sure whether fyne accepts it or not as this doesn't have an appropriate font:

[![Screenshot from Gyazo](https://gyazo.com/a7093cdb96832afcd5ef14e5c73187fc/raw)](https://gyazo.com/a7093cdb96832afcd5ef14e5c73187fc)

**Comment by andydotxyz** [comment]
Apologies we have not completed the unicode support. There is a work in progress (PR #56) which you could perhaps test if you are interested.
Otherwise we will revisit this issue once that new code is landed.

**Comment by hajimehoshi** [comment]
Tried to execute `hello` but failed:

```
hajimehoshi@Hajimes-MacBook-Pro ~/fyne/cmd/hello 
$ go run .
# fyne.io/fyne/driver/gl
../../driver/gl/gl_darwinfix.go:10:5: warning: 'NSOpenGLContext' is deprecated: first deprecated in macOS 10.14 - Please use Metal or MetalKit. [-Wdeprecated-declarations]
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/AppKit.framework/Headers/NSOpenGL.h:202:12: note: 'NSOpenGLContext' has been explicitly marked deprecated here
# fyne.io/fyne/driver/gl
../../driver/gl/gl.go:158:17: undefined: oksvg.ReadIconStream
```

**Comment by hajimehoshi** [comment]
OK this was my mistake: `oksvg` in my `go.mod` was old.

**Comment by hajimehoshi** [comment]
[![Screenshot from Gyazo](https://gyazo.com/7bef72ebcf20f79b14238c97ef6270ed/raw)](https://gyazo.com/7bef72ebcf20f79b14238c97ef6270ed)

I think rendering is not perfect, but it worked. Thank you!

**Comment by goodxp** [comment]
I run the demo on Mac, tried to alter the welcome title by simply adding 2 Chinese characters, looks like it does not support internationalization yet. I wish it could be added in the future. 

![Screenshot 2019-10-11 at 8 25 28 PM](https://user-images.githubusercontent.com/5576304/66651373-75940e00-ec65-11e9-8819-b473a54c4686.png)

![Screenshot 2019-10-11 at 8 25 50 PM](https://user-images.githubusercontent.com/5576304/66651419-880e4780-ec65-11e9-9f24-0032774870a3.png)

**Comment by andydotxyz** [comment]
Can you please try the suggestion higher in this ticket? https://github.com/fyne-io/fyne/issues/63#issuecomment-455850180
Unicode is supported, but we don’t bundle font files for all languages yet.

**Comment by goodxp** [comment]
The higher is still the same. I think it will partially work with Japanese since there are 2 kinds of characters in Japanese, one is like English(a character set with about twice the number in ASCII) and #63 might have fixed it with #63, another kind is like traditional Asian characters like Chinese with a large character set, each character with 2+ bytes presentation in Unicode , which did not be fixed.  This may cause a larger code change - probably need to alter all font drawing related "string" type to []rune and iterate with rune instead of byte.  It is just my guess. Really hope to see this fixed soon.

**Comment by andydotxyz** [comment]
Can you please provide a code example of what does not work? We have had reports that Chinese is working fine after loading the right font so I assumed multibyte was working. We have attempted to use []rune under the hood so if we’ve missed somewhere this should get fixed
